sdk/core/azure-core/inc/azure/core/case_insensitive_map.hpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Add this to the simplified core.hpp header + test.

Reply:
I am adding it in #1752.